ironfish/src/node.ts

```diff
@@ -102,6 +107,7 @@ export class IronfishNode {
       chain: chain,
       strategy: strategy,
       metrics: this.metrics,
+      peerAddressManager: new PeerAddressManager(this.hosts),
```

Comment:
Why does the node keep track of the hosts? What does it use it for? Is there other systems later besides networking that will use the hosts?